osmnx/folium.py

```diff
@@ -122,12 +127,13 @@ def plot_graph_folium(
             edge_width=edge_width,
             edge_opacity=edge_opacity,
             popup_attribute=popup_attribute,
+            **kwargs,
         )
         pl.add_to(graph_map)
 
     # if fit_bounds is True, fit the map to the bounds of the route by passing
     # list of lat-lng points as [southwest, northeast]
-    if fit_bounds:
+    if fit_bounds and type(graph_map) is folium.Map:
```

Comment:
Can you change the 4 `type(var) is datatype` checks into `isinstance(var, datatype)` checks instead?

Reply:
yes